README.md

```diff
@@ -1 +1,201 @@
+
+- [Live Demo Link](https://peter-mwovi.github.io/portfolio_setup/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ npm install
+ ```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+    cd portfolio_setup
+    git clone https://github.com/peter-mwovi/portfolio_setup.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd portfolio_setup
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+    npm run build (it will create a production ready file)
+    npm start (it will run the server on your machine)
+```
+
+### Run 
+
+I have added linters to ensure proper practice in the project.
+
+To run the linter tests on ``` index.html ```,  run the following command:
+
+```sh
+    npx hint .
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+```sh
+    https://vercel.com/
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Peter MWovi**
+
+- GitHub: [@githubhandle](https://github.com/peter-mwovi/)
+- Twitter: [@twitterhandle](https://twitter.com/mwovi_peter)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/peter-mwovi-57141a179/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Web Version**
+- [ ] **Complete Mobile versio**
+- [ ] **Fully responsive**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please follow me on my social media handles then go ahead and contribute.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for providing me with the necessary tools to work on this a project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ  -->
+
+## ❓ FAQ <a name="faq"></a>
+
+
+- **What is Linters?**
+
+  - Linters is a tool to analyse Source code so as to flag errors.
+
+- **Do I need knowledge in javascript to contribute?**
+
+  - basic knowledge in javascript is required. However feel free to play around with the source code.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] The link to your `MIT LICENCE` is broken. Please use these [guidelines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to correctly add LICENCE into your README.